test/successful-files.test.js

```diff
@@ -24,7 +24,7 @@ function makeFileTest(file) {
     try {
       var result = loadConfigSync(null, filePath);
       doAsserts(assert, result, filePath);
-      loadConfig(null, absolutePath(file)).then(function (result) {
+      loadConfig(null, filePath).then(function (result) {
```

Comment:
Actually `filePath` is the same as `absolutePath(file)` and is evaluated at the beginning of the higher order function([`test/successful-files.test.js#L19`](https://github.com/sudo-suhas/cosmiconfig/blob/f393c95ac789cc4ceb9c284d3d16fc8f5b46772e/test/successful-files.test.js#L19)).